### ISSUE TITLE
Rule proposal: prefer-binding-declarations

### ISSUE THREAD

**Issue by robertsspaceindustries** [sindresorhus/eslint-plugin-unicorn#2206]
### Description

Prefer binding lists for declarations for nicer code and less repeated `let`/`var`/`const` declarations.

### Fail

```js
let foo = true; let bar = false;
```

### Pass

```js
let foo = true, bar = false;
```
